### ISSUE TITLE
Error in send1msg API documentation markup

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5656]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5656|
|Type|defect|
|Created|2012-04-26 13:51:02Z|
|Branch|https://github.com/twisted/twisted/tree/send1msg-docstring-5656|

The docstring for `twisted.python.sendmsg.send1msg` has a mistake, it says "@params" instead of "@param" in one place.

This is a pydoctor error, but it was missed because we haven't been building extension module documentation with pydoctor (now we are).

<details><summary>Searchable metadata</summary>

```
trac-id__5656 5656
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_send1msg_docstring_5656 branches/send1msg-docstring-5656
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1335448262000000 1335448262000000
changetime__1335454290000000 1335454290000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [34289]) Branching to 'send1msg-docstring-5656'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [34290]) Spell the markup right

refs [#5656](https://github.com/twisted/twisted/issues/5656)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

[trunk doc build results](http://buildbot.twistedmatrix.com/builders/documentation/builds/2088/steps/api-documentation/logs/pydoctor%20errors)

[branch doc build results](http://buildbot.twistedmatrix.com/builders/documentation/builds/2089/steps/api-documentation/logs/pydoctor%20errors)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Thanks, this looks good. I also noticed you missed an href in r34288:

```
<h2>POSIX Specific Tricks</h2> 
<ul> 
<li><a href="">sendfd.py</a>
```

Please merge after that.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

Oh and while you're at it:

 * `FileDescriptorOverrun`
 * `IFileDescriptorReceiver`
 * `IUNIXTransport`

need @since markers.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

Those comments seem unrelated to this ticket.  They would have made good review comments on [#5615](https://github.com/twisted/twisted/issues/5615).  Since we missed that, I guess they'll make a good basis for a new ticket.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [34295]) Merge send1msg-docstring-5656

Author: exarkun
Reviewer: thijs
Fixes: [#5656](https://github.com/twisted/twisted/issues/5656)

Fix broken markup in `twisted.python.sendmsg` API documentation.
